posthog/warehouse/data_load/service.py

```diff
@@ -50,6 +52,59 @@ def get_sync_schedule(external_data_schema: "ExternalDataSchema"):
 
     sync_frequency, jitter = get_sync_frequency(external_data_schema)
 
+    # format 15:00:00 --> 3:00 PM UTC | default to midnight UTC
+    hour = 0
+    minute = 0
+    if external_data_schema.sync_time_of_day:
+        sync_time_of_day = str(external_data_schema.sync_time_of_day)
+        hour = int(sync_time_of_day.split(":")[0])
+        minute = int(sync_time_of_day.split(":")[1])
```

Comment:
What is the type of `external_data_schema.sync_time_of_day`? Can we not do some smarter parsing than splitting by colon?

Reply:
It's a string which is not ideal, but I opted for some more intelligence with it regardless.